spaceship/lib/spaceship/portal/invite.rb

```diff
@@ -0,0 +1,43 @@
+module Spaceship
+  module Portal
+    class Invite < PortalBase
+      # @return (String) Invite Id
+      attr_accessor :invite_id
+
+      # @return (String) Inviter name
+      attr_accessor :inviter_name
+
+      # @return (String) Recipient Email Address
+      attr_accessor :email_address
+
+      # @return (String) Role (member, admin or agent)
+      attr_accessor :type
+
+      # @return (String) Invite creation date
+      attr_accessor :created
+
+      # @return (String) Invite expiration date
+      attr_accessor :expires
+
+      attr_mapping(
+        'inviteId' => :invite_id,
+        'inviterName' => :inviter_name,
+        'recipientEmail' => :email_address,
+        'recipientRole' => :type,
+        'dateCreated' => :created,
+        'dateExpires' => :expires
+      )
+
+      class << self
+        def factory(attrs)
+          # rubocop:disable Style/RescueModifier
```

Comment:
It would be great if we can avoid using inline `rescue` without an error type specification. Is `attrs['dateCreated']` `nil` sometimes? If so, can we check of `nil` values instead, or catch only the exception type we expect here?